extensions/2.0/Vendor/EXT_feature_metadata/examples/weather/weather.gltf

```diff
@@ -0,0 +1,231 @@
+{
+  "asset": {
+    "version": "2.0"
+  },
+  "extensionsUsed": [
+    "EXT_feature_metadata"
+  ],
+  "extensions": {
+    "EXT_feature_metadata": {
+      "extras": {
+        "draftVersion": "0.0.0"
+      },
+      "featureClasses": {
+        "weather": {
+          "name": "Weather Data",
+          "semantic": "WEATHER",
+          "properties": {
+            "airTemperature": {
+              "name": "Air Temperature",
+              "semantic": "AIR_TEMPERATURE",
+              "elementType": "FLOAT32"
+            },
+            "airPressure": {
+              "name": "Air Pressure",
+              "semantic": "AIR_PRESSURE",
+              "elementType": "FLOAT32"
+            },
+            "windVelocity": {
+              "name": "Wind Velocity",
+              "semantic": "WIND_VELOCITY",
+              "elementType": "ARRAY",
+              "componentType": "FLOAT32",
+              "componentsPerElement": 3
+            }
+          }
+        },
+        "town": {
+          "name": "Town Data",
+          "semantic": "TOWN",
+          "properties": {
+            "name": {
+              "name": "Name",
+              "semantic": "NAME",
+              "elementType": "STRING"
+            },
+            "population": {
+              "name": "Population",
+              "semantic": "POPULATION",
+              "elementType": "FLOAT32"
+            }
+          }
+        }
+      },
+      "featureTables": {
+        "weatherTable": {
+          "featureClass": "weather",
+          "featureCount": 1000,
+          "properties": {
+            "airTemperature": {
+              "elementCount": 1000,
```

Comment:
`"elementCount"` should be moved one level above (or removed since it's the same as `"featureCount"`)

Reply:
ah right I must have missed a spot. Since weather is generated by a script, I might have updated the glTF but forgotten to update the .js file.